### ISSUE TITLE
Animation not playing if `activateAR` is called without `dismissPoster`

### ISSUE THREAD

**Issue by ryancrunchi** [google/model-viewer#2338]
### Description

I have a custom model-viewer page. The model-viewer ar-modes is set to `webxr scene-viewer quick-look`. The reveal attribute is set to `manual`.
I don't use poster or ar-button attributes or slots. Instead I have custom design, with custom button. I don't want to show 3D or AR before user interaction.
When I click this button I call this function :
```javascript
if (modelViewer.canActivateAR) {
    modelViewer.activateAR();
} else {
    modelViewer.dismissPoster(); // if AR is not available dismiss poster so it displays 3D anyway
}
```
In this case, webxr starts but model is not animated.

When I change to :
```javascript
// dismissPoster will make animations start in AR. If AR is not available dismiss poster will display 3D anyway
modelViewer.dismissPoster();
if (modelViewer.canActivateAR) {
    modelViewer.activateAR();
    modelViewer.showPoster(); // hide the 3D back so when the AR is closed you don't see 3D.
}
```
Here the animations start. But when I close webxr I have the 3D left visible.
Workaroud : use `ar-status` event to call `modelViewer.showPoster()` to hide 3D when status is `not-presenting`
But, when the AR mode falls back to SceneViewer I don't have event to tell that AR was closed and hide 3D.

### Version

- model-viewer: v1.6.0

### Browser Affected

- [x] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [x] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [x] WebXR
- [ ] SceneViewer
- [ ] QuickLook

**Comment by timmmeh** [comment]
Can you provide a repro link?

**Comment by elalish** [comment]
Can you put a link where I can repro? I've been refactoring the webXR render path a bit lately; is it fixed on master by chance?

**Comment by ryancrunchi** [comment]
Problem is solved with version 1.7.1
